android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java

```diff
@@ -114,6 +117,13 @@ public void setResizeMode(FastImageViewWithUrl view, String resizeMode) {
         view.setScaleType(scaleType);
     }
 
+    @ReactProp(name = "resizeImageAndroid")
+    public void setImageResize(FastImageViewWithUrl view, ReadableMap imageSizeOverride) {
+        // Re-run Glide with width and height override values set.
```

Comment:
Wouldn't it be better to put `imageResize` into source, so we don't have to rerun Glide? It would be beneficial for iOS as well.